eZ/Publish/Core/FieldType/RichText/XSLTProcessorFunctions.php

```diff
@@ -12,19 +12,50 @@
  */
 class XSLTProcessorFunctions
 {
-    const URL_PATTERN = '%^((ezlocation|ezurl|ezcontent):\/\/\d+(#[^\s]+)?)|(#[^\s]+)|(?:(?:https?|ftps?)://)(?:\S+(?::\S*)?@|\d{1,3}(?:\.\d{1,3}){3}|(?:(?:[a-z\d\x{00a1}-\x{ffff}]+-?)*[a-z\d\x{00a1}-\x{ffff}]+)(?:\.(?:[a-z\d\x{00a1}-\x{ffff}]+-?)*[a-z\d\x{00a1}-\x{ffff}]+)*(?:\.[a-z\x{00a1}-\x{ffff}]{2,6}))(?::\d+)?(?:[^\s]*)?$%iu';
+    const URL_SCHEMA_WHITELIST = [
+        'http',
+        'https',
+        'ftp',
+        'ftps',
+        'ezcontent',
+        'ezlocation',
+        'ezurl',
+    ];
```

Comment:
We might include `mailto:`, ref suggestion in https://github.com/ezsystems/ezpublish-kernel-ee/pull/161